### ISSUE TITLE
Change Elastic mapping type of serde_json::Value

### ISSUE THREAD

**Issue by lfdominguez** [elastic-rs/elastic#411]
The default mapping type for serde_json::Value is "nested" and that not allow make normal queries on ElasticSearch for dynamics data. How i can change the type to "object". Already i changed the code on file "src/elastic/src/types/document/mapping.rs" on trait `ObjectMapping` from `NESTED_DATATYPE` to `OBJECT_DATATYPE` and now my elastic queries are responding with results.

**Comment by mwilliammyers** [comment]
You can also do it though Elasticsearch [directly](https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping.html), I use [dynamic mappings](https://www.elastic.co/guide/en/elasticsearch/reference/current/dynamic-mapping.html) personally...

**Comment by lfdominguez** [comment]
> You can also do it though Elasticsearch [directly](https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping.html), I use [dynamic mappings](https://www.elastic.co/guide/en/elasticsearch/reference/current/dynamic-mapping.html) personally...

Yes of course, i can use curl directly to put data on ElasticSearch too. The main question is why use "nested" for object types and not "object"
